sdk/react/ooyalaSkinPanelRenderer.js

```diff
@@ -125,7 +125,7 @@ export default class OoyalaSkinPanelRenderer {
           castControls: this.skin.props.castControls,
           icons: this.skin.props.icons,
         }}
-        selectedDeviceId={this.skin.state.connectedDevice.id}
+        selectedDeviceId={this.skin.state.connectedDevice === undefined ? '' : this.skin.state.connectedDevice.id}
```

Comment:
It's quite unsafe. You can convert it to boolean: `!this.skin.state.connectedDevice ? '' : this.skin.state.connectedDevice.id`

Reply:
done